fabric_cf/actor/fim/plugins/broker/aggregate_bqm_plugin.py

```diff
@@ -305,12 +307,14 @@ def plug_produce_bqm(self, *, cbm: ABCCBMPropertyGraph, **kwargs) -> ABCBQMPrope
                         fp_cpoint_props = {ABCPropertyGraph.PROP_NAME: parent_props[ABCPropertyGraph.PROP_NAME] + '-' +
                                                                        fac_port_props[ABCPropertyGraph.PROP_NAME],
                                            ABCPropertyGraph.PROP_TYPE: peer_props[ABCPropertyGraph.PROP_TYPE],
-                                           ABCPropertyGraph.PROP_CAPACITIES: peer_props[ABCPropertyGraph.PROP_CAPACITIES],
-                                           ABCPropertyGraph.PROP_LABELS: peer_props[ABCPropertyGraph.PROP_LABELS]}
+                                           ABCPropertyGraph.PROP_CAPACITIES:
```

Comment:
NetworkX doesn't like None fields, so we would have to add a conditional check on not None values to be added.